src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,33 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        Direction targetDirectionX = robot.getDirection();
```

Comment:
lets try to solve without this variable and same at line 30

Reply:
done